src/reanimated2/component/FlatList.tsx

```diff
@@ -78,13 +81,23 @@ export const ReanimatedFlatList = forwardRef(
       [cellStyle]
     );
 
-    return (
+    const animatedFlatList = (
       <AnimatedFlatList
         ref={ref}
         {...restProps}
         CellRendererComponent={cellRenderer}
       />
     );
+
+    if (skipLayoutAnimations === undefined) {
+      return animatedFlatList;
+    }
+
+    return (
```

Comment:
I'm not a fan of this implementation. As discussed, this makes our API inconsistent. Every Animated component has to be wrapped in the skipper component, except for FlatList. Then I guess we would have multiple users both using this setting and wrapping it manually.